bigbluebutton-html5/imports/ui/components/chat/component.jsx

```diff
@@ -31,7 +36,16 @@ export default class Chat extends Component {
         <header className={styles.header}>
           <Link className={styles.closeChat} to="/users">
             <Icon iconName="left-arrow" /> {title}
+            <Button className={styles.closeBtn}
```

Comment:
> **and remove the chat from the sidebar.**

I dont think its a good idea to implement it half-way "silently" on some random PR.